pages/setup/account.page.tsx

```diff
@@ -0,0 +1,146 @@
+import { GetServerSideProps } from 'next';
+import Head from 'next/head';
+import { useRouter } from 'next/router';
+import React, { useState } from 'react';
+import { Autocomplete, TextField } from '@mui/material';
+import { getSession } from 'next-auth/react';
+import { useTranslation } from 'react-i18next';
+import {
+  UpdateUserDefaultAccountDocument,
+  UpdateUserDefaultAccountMutation,
+  UpdateUserDefaultAccountMutationVariables,
+  useUpdateUserDefaultAccountMutation,
+} from 'src/components/Settings/preferences/accordions/DefaultAccountAccordion/UpdateDefaultAccount.generated';
+import { SetupPage } from 'src/components/Setup/SetupPage';
+import { LargeButton } from 'src/components/Setup/styledComponents';
+import useGetAppSettings from 'src/hooks/useGetAppSettings';
+import makeSsrClient from 'src/lib/apollo/ssrClient';
+import {
+  AccountListOptionsDocument,
+  AccountListOptionsQuery,
+} from './Account.generated';
+
+type AccountList = AccountListOptionsQuery['accountLists']['nodes'][number];
+
+interface PageProps {
+  accountListOptions: AccountListOptionsQuery;
+}
+
+// This is the third page page of the setup tour. It lets users choose their default account list. It will be shown if
+// the user has more than one account list and don't have a default chosen yet.
+const AccountPage: React.FC<PageProps> = ({ accountListOptions }) => {
+  const { t } = useTranslation();
+  const { appName } = useGetAppSettings();
+  const { push } = useRouter();
+  const [updateUserDefaultAccount, { loading: isSubmitting }] =
+    useUpdateUserDefaultAccountMutation();
+
+  const [defaultAccountList, setDefaultAccountList] =
+    useState<AccountList | null>(null);
+
+  const handleSave = async () => {
+    if (!defaultAccountList) {
+      return;
+    }
```

Comment:
Show have some kind of error message to tell the user they need to select a default account list